packages/translations/src/i18next/i18next.js

```diff
@@ -140,11 +140,4 @@ const loadIncontextTranslation = () => {
     }
 };
 
-export default {
-    changeLanguage,
-    getAllLanguages,
-    getLanguage,
-    initializeTranslations,
-    localize,
-    Localize,
-};
+export { changeLanguage, getAllLanguages, getLanguage, initializeTranslations, localize, Localize };
```

Comment:
Can we just add the `export` keyword to each of the declarations instead? I might be wrong, but I believe there is a difference between the two going back to tree shaking*, and object export/destructuring vs. named export/import.

*(which I may be wrong, because it might happen either way)

Reply:
I'm not sure if there is a difference between `export function ...` and `export { function1, function2 }` for tree shaking  but can change it just in case.